components/authz-service/storage/postgres/migration/migration.go

```diff
@@ -63,7 +63,9 @@ func (c *Config) Migrate(dataMigConf datamigration.Config) error {
 	}
 
 	if version < PreForceUpgradeMigration {
+		l.Infof("Migrating schema from version %v to %v...", version, PreForceUpgradeMigration)
 		err = m.Migrate(PreForceUpgradeMigration)
+		l.Infof("Migrating schema from version %v to %v -- complete", version, PreForceUpgradeMigration)
```

Comment:
I think we'll want this after the err check

Reply:
Hmm... tried that and it makes it more complicated. Have to have one log statement for completed successfully, then another for completed with error since the err-check does a return. Then same changes on other blocks of code, where some have multiple error checks... So I am inclined to leave it as is.

Reply:
Hmm, I'm happy to have a go at it myself since I don't totally understand the scope of the problem you're mentioning. It looks to me like you could add a log statement before the err return along the lines of "migration to 74 failed" and then the "complete" log after that. 

But at a minimum, I think the phrase "complete" is misleading here, as it may have failed, so if we can't move it, I'd opt to delete it.